source/extensions/filters/common/local_ratelimit/local_ratelimit_impl.cc

```diff
@@ -48,6 +48,7 @@ LocalRateLimiterImpl::LocalRateLimiterImpl(
       throw EnvoyException(
           "local rate descriptor limit is not a multiple of token bucket fill timer");
     }
+    new_descriptor.multiplier_ = token_bucket.fill_interval_ / token_bucket_.fill_interval_;
```

Comment:
Can you add a comment here? Reading this quickly it's confusing what this is doing.